packages/webpack-cli/package.json

```diff
@@ -22,6 +22,9 @@
     "bundler",
     "web"
   ],
+  "files": [
+    "lib"
```

Comment:
also need `bin`

Reply:
Nope, we have the `"bin"` field, what mean automatically adding "bin" directories to packages, try to run `npm pack`/`yarn pack`

Reply:
Bug on rule side :smile:

Reply:
Let's adding it here and I will report about this bug

Reply:
👍